src/module/migration/migrations/885-convert-alignment-damage.ts

```diff
@@ -76,7 +79,7 @@ export class Migration885ConvertAlignmentDamage extends MigrationBase {
             }
         }
 
-        fu.mergeObject(source.system.attributes, iwr);
```

Comment:
If the ?? occurs, there won't be any changes since its not being assigned to anything.

Reply:
![image](https://github.com/user-attachments/assets/ff8ce509-c28f-4646-8cf4-76125636f662)

Reply:
If the actor already (somehow) has no attributes, that's fine.

Reply:
No, what I mean is that I think you have to do `source.system.attributes = fu.mergeObject(source.system.attributes ?? {}, iwr);`. If ?? {} occurs, its an entirely new object which is not assigned to anything. You might as well do an if statement on source.system.attributes since nothing is being mutated.